File: 1603-Parking-System/1603.java
class ParkingSystem {
  int[] cartContainer;
  public ParkingSystem(int big, int medium, int small) {
    cartContainer = new int[]{big, medium, small};
  }

  public boolean addCar(int carType) {
    return cartContainer[carType - 1]-- > 0;
  }
}

File: 1603-Parking-System/1603.py
class ParkingSystem:
  def __init__(self, big: int, medium: int, small: int):
    self.num_dic = {1:big,2:medium,3:small}

  def addCar(self, carType: int) -> bool:
    if self.num_dic[carType] > 0:
      self.num_dic[carType] -= 1
      return True
    return False